Detections/ZoomLogs/JoiningMeetingFromAnotherTimeZone.yaml

```diff
@@ -0,0 +1,40 @@
+id: 58fc0170-0877-4ea8-a9ff-d805e361cfae
+name: User joining Zoom meeting from suspicious timezone
+description: |
+  'The alert shows users that join a Zoom meeting from a time zone other than the one the meeting was created in.
+  You can also whitelist known good time zones in the tz_whitelist value using the tz database name format https://en.wikipedia.org/wiki/List_of_tz_database_time_zones'
+severity: Low
+requiredDataConnectors:
+  - connectorId: CustomConnector
+    dataTypes:
+      - ZoomLogs
+queryFrequency: 6h
+queryPeriod: 6h
```

Comment:
Default 1d

Reply:
Actually, this should be 14d, we don't want to use 30d lookbacks due to potential perf hits